helm/kubernetes-nginx-ingress-controller-chart/templates/controller-hpa.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.global.hpa.enabled }}
```

Comment:
I don't think we need to use global. This was hacky and due to the k8scc migration. As we needed it to install the temp resources.

We still need to simplify the templates but because the IC migration is disabled for all providers I think it can be removed.

Reply:
ok I do an extra section